build

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#
+# Build steps to validate the ketohub raw input spider.
+
+# Run unit tests and calculate code coverage.
+coverage run \
+  --source ketohub \
+  -m unittest discover &&
+# Check that source has correct formatting.
+yapf --diff --recursive --style google ketohub/spiders/*.py &&
```

Comment:
We want all of `ketohub/` and `tests/` rather than just `ketohub/spiders/` right? Ditto for the next two commands.